examples/docker-mixin-test/README.md

```diff
@@ -1,43 +1,30 @@
-# My First Bundle
+# Docker Mixin Example Bundle
 
-This is an empty bundle that porter has created to get you started!
+This is an example bundle to show how to use the Docker mixin in Porter!
 
 # Contents
 
 ## porter.yaml
 
 This is the porter manifest. See https://porter.sh/author-bundles/ for 
 details on every field and how to configure your bundle. This is a required
-file.
+file. 
 
-## helpers.sh
+The commands available in the docker mixin are docker pull, push, build, 
+run, remove, and login. 
 
-This is a bash script where you can place helper functions that you can call
-from your porter.yaml file.
+### Notes on docker login
+Login has two optional parameters - username and password. To pass in the 
+username and password, you can either enter them in the yaml or set them 
```

Comment:
Let's leave out "you can set them in the yaml" as that isn't what we recommend, embedding sensitive credentials in the yaml. So the text should instead just send them down the recommended path.